.build/config/README.md

```diff
@@ -0,0 +1,125 @@
+Declarative Test Suite Configuration
+-------------------------------------------
+
+Pipeline and test wuite configurations are declarative so other CI implementations can build 
```

Comment:
@jmckenzie-dev  wuite -> suite

Reply:
👍 Thanks for the catch. Fixed and will need to rewrite the file in general after we settle on final .yaml format.